sstables/sstables.cc

```diff
@@ -2392,14 +2392,6 @@ entry_descriptor entry_descriptor::make_descriptor(sstring sstdir, sstring fname
     return make_entry_descriptor(std::move(sstdir), std::move(fname), &ks, &cf);
 }
 
-sstable::version_types sstable::version_from_sstring(sstring &s) {
```

Comment:
I use this function in #12523 [here](https://github.com/scylladb/scylladb/pull/12523/commits/f957d76e3da18357840c295da1b8461994fe582e) so I'll have to revert this patch in my final PR (hopefully next week)
@denesb

Reply:
ack. thanks for the heads-up.

Reply:
You can still dequeue it, it wasn't promoted yet.

Reply:
Yup, dequeued

Reply:
@tchaikov can you please close this PR manually.

Reply:
@xemul @denesb thanks you! i dropped the first commit from this patchset.